src/data-access/repositories/mongo/mongo-rundown-repository.ts

```diff
@@ -38,10 +38,13 @@ export class MongoRundownRepository extends BaseMongoRepository implements Rundo
 
   public async getRundown(rundownId: string): Promise<Rundown> {
     this.assertDatabaseConnection(this.getRundown.name)
-    await this.assertRundownExist(rundownId)
-    const mongoRundown: MongoRundown = (await this.getCollection().findOne({
+    const mongoRundown: MongoRundown | null = (await this.getCollection().findOne<MongoRundown>({
```

Comment:
The outermost parentheses can be removed

Reply:
It almost feel like you have a personal agenda against parentheses ;)
They have been removed.